src/renderer_d3d12.cpp

```diff
@@ -844,13 +844,31 @@ namespace bgfx { namespace d3d12
 
 								if (SUCCEEDED(hr))
 								{
-//									debug1->SetEnableGPUBasedValidation(true);
+									//This is the least ugly way to get the windows build that still works past win10 without deprecation warnings or needing app manifests
+									RTL_OSVERSIONINFOW osver;
```

Comment:
Move this into function, make content of function ` #if BX_PLATFORM_WINDOWS` because D3D1x compiles on Linux too.